openpype/hosts/houdini/plugins/create/create_review.py

```diff
@@ -56,30 +57,48 @@ def create(self, subset_name, instance_data, pre_create_data):
                 "aspect": pre_create_data.get("aspect"),
             })
 
+        lop_enabled = pre_create_data.get("lopEnabled")
```

Comment:
Can't we just stick close to what Houdini uses internally and define a `opSource` or `source` type of data which defines either "SOPs" or "LOPs". That would also leave the path open to potential other sources that might ever appear in the future in Houdini? Plus it's also clearer that when LOPs is not chosen that it uses SOPs.